src/js/models/appUser.js

```diff
@@ -42,7 +42,7 @@ var AppUser = module.exports = BaseModel.extend({
             return Backbone.Model.prototype.save.apply(this, _.toArray(arguments)).then(function() {
                 this._lastPropertyValues = this.pick(AppUser.EDITABLE_PROPERTIES);
                 return arguments;
-            });
+            }.bind(this));
```

Comment:
Yeah that's what I meant when I said we aren't supporting crumbling browsers. I thought you meant IE 8 :tongue: